core/src/main/kotlin/apiDocs/ProductsDocs.kt

```diff
@@ -430,7 +430,8 @@ val deleteUserFromProductGroup: OpenApiRoute.() -> Unit = {
 
 val getVulnerabilitiesAcrossRepositoriesByProductId: OpenApiRoute.() -> Unit = {
     operationId = "GetVulnerabilitiesAcrossRepositoriesByProductId"
-    summary = "Get the vulnerabilities from latest successful advisor runs across the repositories in a product."
+    summary = "Get vulnerabilities from a product"
+    description = "Get the vulnerabilities from latest successful advisor runs across the repositories in a product."
```

Comment:
Maybe: "Get the vulnerabilities from last successful runs for each repository in a product."